ignite/distributed/utils.py

```diff
@@ -386,8 +386,8 @@ def broadcast(
         src: source rank. Default, 0.
         safe_mode: if True, non source input data can be ``None`` or anything (will be discarded), otherwise data
             type of the input ``tensor`` should be respected for all processes. Please, keep in mind, this mode is
-            working only for dense tensors as source input if a tensor is provided. There are additional collective
-            ops are performed before doing the broadcast and, thus, can be slower than without using this mode.
+            working only for dense tensors as source input if a tensor is provided. With ``safe_mode=False``, some
```

Comment:
@sadra-barikbin the way how you updated the docstring does not seem good. With safe mode false, there are no additional collective ops performed... can you explain what's wrong with original text ?

Reply:
The original typo was `There are additional collective ops are performed`, but I wanted to rephrase the sentence as well.

Reply:
And what's wrong with this ?

Reply:
It is written `There are something are performed` that's grammatically wrong.

Reply:
I can keep the sentence as it was but just omit the second `are`.